### PR TITLE
GH #2385: Missing Wiris config caused CKEditor crash in create/edit NDLA mode

### PR DESCRIPTION
**Pull request by chrieinv**
- Add missing settings from adapter in create/edit H5P config
- Fix NDLA adapter accessing protected variable in H5P config
- Update unittests

### CODE REVIEW

File: sourcecode/apis/contentauthor/app/Libraries/H5P/H5PConfigAbstract.php [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
 abstract class H5PConfigAbstract implements ConfigInterface
 {
-    protected const CACHE_BUSTER_STRING = '2.0.3';
+    public const CACHE_BUSTER_STRING = '2.0.3';
     protected const EMBED_TEMPLATE = '<iframe src="%s" width=":w" height=":h" frameborder="0" allowfullscreen="allowfullscreen" allow="geolocation *; microphone *; camera *; midi *; encrypted-media *" title="%s"></iframe>';
 
     public function __construct(
-        protected H5PAdapterInterface $adapter,
+        public H5PAdapterInterface $adapter,
```
Comment (emmachughes) [inline sourcecode/apis/contentauthor/app/Libraries/H5P/H5PConfigAbstract.php:21]: Why the change to public?

Comment (chrieinv) [reply sourcecode/apis/contentauthor/app/Libraries/H5P/H5PConfigAbstract.php:21]: Used it in the tests to verify that the correct Adapter was set, and forgot to revert it

**Review by emmachughes — APPROVED** [review]

**Review by ssethupathy — APPROVED** [review]
